### DISCUSSION TITLE
Problem beim Update von Version 3.53.30.20201024

### DISCUSSION THREAD

**Discussion by tampara** [OpenCCU/OpenCCU#1201] [🇩🇪 Fragen&Antworten] [answered]
Guten Morgen,

folgende Situation ist hier gegeben:

Raspberry PI 3 mit 128 GB MicroSD Karte
HM-MOD-RPI-PCB
RaspberryMatic Version 3.53.30.20201024
Ein Upgrade auf Version 3.57.4.20210320 via Web-Interface lässt sich normal installieren. Nach einem Neustart fährt der Pi allerdings nicht mehr hoch (circa 2 Stunden gewartet).

Eine frische MicroSD-Karte mit dem RaspberryMatic-Image 3.57.4.20210320 lässt den Pi auch nicht starten. Die LED leuchtet nur rot, keine weitern Aktivitäten ersichtbar. Auch das LAN-Interface startet am Pi nicht (keine LEDs sichtbar).

Nur mit der Version 3.53.30.20201024 startet das Pi einwandfrei - inkl. LAN-Interface.

Liegt hier der Fehler zwischen den Ohren bzw. könnt ihr mich unterstützen?

Vielen Dank für eure Hilfe.
Stefan

**Comment by Baxxy13** [comment, marked as answer]
> Liegt hier der Fehler zwischen den Ohren bzw. könnt ihr mich unterstützen?

Bist du dir ganz sicher das es ein Raspberry Pi3B / Pi3B+ ist?

Vom Gefühl her würde ich sagen du verwendest einen Pi2B. 
Mit der RaspberryMatic 3.53.34.20201121 (eine Version nach Deiner) gab es einen "Strukturwechsel" auf 64bit. Seit dem gibt es auch eine separate Version für die 2er Pi's, mit der üblichen 3er Pi Version starten die 2er Pi's dann nicht.

Also mal bitte die Modellbezeichnung deines Raspberry Pi genau anschauen.
